.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  "extends": ["airbnb-base"],
+  "parserOptions": {
+    "ecmaVersion": 8,
+    "sourceType": "module"
+  },
+  "rules": {
+    "global-require": ["off"],
+    "comma-dangle": ["error", "never"],
+    "no-plusplus": ["off"],
+    "arrow-parens": ["off"],
+    "no-restricted-syntax": ["off"],
+    "no-console": ["off"],
+    "no-continue": ["off"],
+    "import/no-unresolved": [2, {}],
```

Comment:
Why is this? I found this super useful

Reply:
This does not turn off this rule. It merely tells the plugin to use the default settings.